### PR TITLE
Add a note about localhost loopback and json api displaying problems

### PR DESCRIPTION
**Pull request by vidmantas**
Related to #313 

Any feedback warmly welcomed - I'm not native English speaker and would love to improve my sentences.

### CODE REVIEW

File: docs/INSTALLING.md [hunk 1]
```diff
@@ -96,6 +96,10 @@ Point your browser (or make a direct request) to `http://api.localhost:49235/use
 }
 ```
 
+If your browser does not support localhost loopback (address not found error), please use `http://api.lvh.me:49235/users` address instead (lvh.me resolves itself and all subdomains to your `localhost` accordingly).
```
Comment (joshsmith) [inline docs/INSTALLING.md]: I would actually just change L88 to say `http://api.lvh.me:49235/users` with the note that:

> lvh.me resolves itself and all subdomains to your `localhost` accordingly

If someone more advanced wants to use `api.localhost`, then they can without probably much need for fruther explanation.

File: docs/INSTALLING.md [hunk 2]
```diff
@@ -96,6 +96,10 @@ Point your browser (or make a direct request) to `http://api.localhost:49235/use
...
+
+Note: some browsers, e.g. Safari, Firefox, can ask to download file instead of displaying its contents directly. This is [known issue for JSON API](https://github.com/json-api/json-api/issues/1048) and if you find this inconvenient, please use Chrome or Opera.
```
Comment (joshsmith) [inline docs/INSTALLING.md]: ", e.g. Safari, Firefox, can" => "like Safari and Firefox may"

"This is a"

Also maybe add recommendation to use tools like https://www.getpostman.com/ or https://paw.cloud/

**Review by joshsmith — CHANGES_REQUESTED** [review]
Thanks for doing this! Just some suggestions for how to improve things here. Let me know if you disagree.

**Comment by joshsmith** [comment]
@vidmantas let me know if you need anything. Always in [Slack](http://slack.codecorps.org) if needed.

**Comment by vidmantas** [comment]
@JoshSmith thanks for help! Could you please take another look to an updated copy?

File: docs/INSTALLING.md [hunk 3]
```diff
@@ -96,6 +96,8 @@ Point your browser (or make a direct request) to `http://api.localhost:49235/use
 }
 ```
 
+Note: some browsers like Safari and Firefox may can ask to download file instead of displaying its contents directly. This is a [known issue for JSON API](https://github.com/json-api/json-api/issues/1048) and if you find this inconvenient, please use Chrome or Opera. We recommend to use specialised tools for API development and discovery like [Postman](https://www.getpostman.com/) or [Paw](https://paw.cloud/).
```
Comment (begedin) [inline docs/INSTALLING.md]: you have a "may can ask you to download" here.

**Review by begedin — CHANGES_REQUESTED** [review]
One more minor change. Should also probably rebase, base branch is out of date.

File: docs/INSTALLING.md (hunk 3 above)
Comment (vidmantas) [reply docs/INSTALLING.md]: Oups, fixed, thanks for pointing out.

**Review by joshsmith — APPROVED** [review]

**Comment by joshsmith** [comment]
@vidmantas looks great! You just need to rebase and squash and you'll be good to go.

**Comment by vidmantas** [comment]
@JoshSmith rebased, but now CI fails when trying to upload the report: 

``` elixir
** (ExCoveralls.ReportUploadError) Failed to upload the report to 'https://coveralls.io' (reason: status_code = 422, body = {"message":"Couldn't find a repository matching this job.","error":true}).
```

Anything I can do in this situation?

**Comment by joshsmith** [comment]
@vidmantas I need to fix this. Stand by.

**Comment by joshsmith** [comment]
@vidmantas you should be able to `rebase` again and it should work.

**Comment by vidmantas** [comment]
@JoshSmith worked, thanks!

**Comment by vidmantas** [comment]
@begedin could you take another look?

File: docs/INSTALLING.md [hunk 4]
```diff
@@ -96,6 +96,8 @@ Point your browser (or make a direct request) to `http://api.localhost:49235/use
 }
 ```
 
+Note: some browsers like Safari and Firefox may ask to download file instead of displaying its contents directly. This is a [known issue for JSON API](https://github.com/json-api/json-api/issues/1048) and if you find this inconvenient, please use Chrome or Opera. We recommend to use specialised tools for API development and discovery like [Postman](https://www.getpostman.com/) or [Paw](https://paw.cloud/).
```
Comment (begedin) [inline docs/INSTALLING.md]: Sorry to be so such a nitpick, but, "...may ask to download **a** file..."

**Review by begedin — CHANGES_REQUESTED** [review]
Almost done. Again, sorry it moves so slowly.

File: docs/INSTALLING.md (hunk 4 above)
Comment (vidmantas) [reply docs/INSTALLING.md]: @begedin absolutely no problem, fixed ;)

**Comment by joshsmith** [comment]
🙌  @vidmantas!

**Pull request merged**